packages/language-markdown/package.json

```diff
@@ -19,10 +19,10 @@
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
     "react-scripts": "^3.2.0",
-    "typescript": "^3.6.2"
+    "typescript": "^3.7"
   },
   "devDependencies": {
-    "typescript": "^3.6.2"
+    "typescript": "^3.8"
```

Comment:
Two different versions of TS defined in the same package.json sounds like it might cause trouble